CHANGELOG.md

```diff
@@ -1,43 +1,57 @@
 # Changelog
+
 ## [Unreleased]
 
 ## [2.0.0] - 2021-06-24
 
 ### Added
+
 - Authentication Test
 - Settings on the configurations tab on the app
 
 ### Fixed
+
 - Removed some unused fields
 
 ### Fixed
+
 - Correção de erro de toggle em workspace master.
 
 ### Fixed
+
 - Criar página do Avatax BR no Admin
 - Modularizar o código do front do Avatax BR
 - Conectar o Avatax-BR-Admin com o Avatax-BR-Service
 - Criar área de "Ativar/Desativar Taxação Avalara
 - Remoção do Auth e criação do aplicativo no settings
 
-### Changed 
+### Changed
+
 - Completar o endereço a partir do CEP
 
-### Changed 
+### Changed
+
 - Adicionado o DropDown no nome da Doca com os valores das Docas existentes
 - Completado o id da Doca após escolher o nome da Doca
 - Arrumado a descrição do cityCode
-- Remoção do Auth e criação do aplicativo no settings 
+- Remoção do Auth e criação do aplicativo no settings
 
 ### Changed
+
 - Transformar o avatax-br-admin em avatax-br
 - Validação dos campos
-1. Campos obrigatórios como request, 
-2. CEP com necessariamente 8 dígitos 
+
+1. Campos obrigatórios como request,
+2. CEP com necessariamente 8 dígitos
 3. Validar se o CNPJ é válido
 4. Toda vez que é preenchidos os valores de endereço vindo do CEP deve-se atualizar para que esses valores não apareçam como o aviso de campo obrigatório
 5. Toda vez que se clica no botão “Adicionar” deve-se rodar a validação e preencher com avisos os campos que tem valores errados
 6. Toda vez que se sair de um campo é necessário fazer a validação do campo e retornar erro caso haja incoerência no campo
 
+### Changed
+
+- Trocar o campos com texto para campos com intl
```

Comment:
Acredito que essa tag de change deveria ficar abaixo da tag `[Unreleased]` da linha 3, pro bot reconhecer que existe uma mudança.

Reply:
Fixed in 45d05fa8d5dda5da961e70e9d14eac971b7219c6